### PR TITLE
Updating runtest.py so that it works with Python 3

### PR DESCRIPTION
**Pull request by tannergooding**
https://github.com/dotnet/coreclr/pull/19213 updated runtest.cmd to start calling runtest.py on Windows. For users which only have python 3 installed, this broke the ability to run tests.

This updates the runtest.py file so that it works under Python 3.

### CODE REVIEW

**Comment by tannergooding** [comment]
CC. @jashook

File: tests/runtest.py [hunk 1]
```diff
@@ -1318,7 +1318,7 @@ def copy_tree(src, dest):
 
                 if host_os != "Windows_NT":
                     # Set executable bit
-                    os.chmod(os.path.join(dest, item), 0774)
+                    os.chmod(os.path.join(dest, item), 774)
```
Comment (tannergooding) [inline tests/runtest.py]: This is the only line that doesn't modify a `print` call

File: tests/runtest.py [hunk 2]
```diff
@@ -1279,9 +1279,9 @@ def setup_core_root(host_os,
                  "-BuildArch=%s" % arch,
                  "-BuildOS=%s" % host_os]
 
-    print ""
```
Comment (tannergooding) [inline tests/runtest.py]: This, L1394, and L873 are the were changed from `print ""` to `print`.

**Comment by tannergooding** [comment]
The rest of the changes were `print args` to `print(args)`

File: tests/runtest.py [hunk 3]
```diff
@@ -1716,7 +1716,7 @@ def parse_test_results(host_os, arch, build_type, coreclr_repo_location, test_lo
     return tests
 
 def print_summary(tests):
-    """ Print a summary of the test results
+    """ print(a summary of the test results
```
Comment (jashook) [inline tests/runtest.py]: This change should not be incldued

Comment (tannergooding) [reply tests/runtest.py]: oops, will fix

**Comment by tannergooding** [comment]
test Windows_NT arm64 Cross Debug Innerloop Build please
test Windows_NT arm Cross Debug Innerloop Build please

**Pull request merged**